src/organisms/layer.py

```diff
@@ -6,6 +7,10 @@ def __init__(self, nodes_in, size):
         self.biases = [0 for node in range(size)] # bias for every node in the layer
     
 
+    def ActivationFunction(x):
```

Comment:
I think here a comment would be necessary (most people don't see directly, what the function does).
Also usually, functions begin with a lowercase letter. Why didn't you do that here?